record/record.go

```diff
@@ -127,11 +131,40 @@ type Reader struct {
 
 // NewReader returns a new reader.
 func NewReader(r io.Reader) *Reader {
+	rs, _ := r.(io.ReadSeeker)
 	return &Reader{
-		r: r,
+		r:  r,
+		rs: rs,
 	}
 }
 
+// Skips all blocks that are completely before offset.
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences says that "Comments should begin with the name of the thing being described and end in a period". Thus:

// SkipToInitialOffset skips all blocks that etc.

Similarly below.